frontend/components/FileDetails/FileDetails.tsx

```diff
@@ -20,6 +23,7 @@ interface IFileDetailsProps {
   onFileSelect?: (e: React.ChangeEvent<HTMLInputElement>) => void;
   accept?: string;
   progress?: number;
+  gomEnabled?: boolean;
```

Comment:
Same her, gom should be spelled out because it's confusing to run into that later in the file without knowing what it is.